messagebus/messagebus.go

```diff
@@ -129,6 +135,17 @@ func (m msgBus) SendMessage(subject string, host string, route config.Route, pri
 	return m.natsConn.Publish(subject, json)
 }
 
+func (m msgBus) mapRouteOptions(route config.Route) map[string]string {
+	if route.Options != nil {
+		routeOptions := make(map[string]string)
+		if route.Options.LoadBalancingAlgorithm != "" {
+			routeOptions[LoadBalancingAlgorithm] = string(route.Options.LoadBalancingAlgorithm)
```

Comment:
if the value doesn't exist you will return an empty map. is that what you want? I suggest to return nil if no value is found for the key.

Reply:
In my understanding, the `options` will not be restricted to `lb_algo` only. 
If no `options` per route exist, nil will be returned.
If `options` exist, but do not contain `lb_algo` (and can contain any other custom properties), the map will be returned. But you are right, it can be then empty map.

Reply:
> But you are right, it can be then empty map.
Yes, that was my point. I suggest to not blindly get whatever key from the map but also verify that a value for the key exists. If there is no value then don't put an empty "lb_alg" key in the map, but rather none at all.

Reply:
It may be a fringe case. We shouldn't see anything else but validated keys here. Resolving.